<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(20, 235, 235);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Home_plus_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_plus_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_plus_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_min_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Home_score">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 0, 255);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Away_score">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 0, 255);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_plus_1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_plus_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_plus_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_min_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Score_reset">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="Timer_display">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_start">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_pause">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_reset">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_set_15_minutes">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>15min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_set_10_minutes">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_set_5_minutes">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_set_3_minutes">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_set_1_minute">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_plus_1_minute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timer_plus_1_second">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1det</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Round">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 21, 21);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="Round_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Round_inc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Round_dec">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Home_foul">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(219, 8, 8);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_foul_dec">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home_foul_inc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Away_foul">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 16, 16);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_foul_inc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Away_foul_dec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Twenty_four">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>85</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>24 detik</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="Twenty_four_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Tet_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>TET</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Home_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Away_name">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fourteen">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>85</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>14 detik</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <zorder>Timer_plus_1_minute</zorder>
   <zorder>Home_plus_1</zorder>
   <zorder>Home_plus_2</zorder>
   <zorder>Home_plus_3</zorder>
   <zorder>Home_min_1</zorder>
   <zorder>Home_score</zorder>
   <zorder>Away_score</zorder>
   <zorder>Away_plus_1</zorder>
   <zorder>Away_plus_2</zorder>
   <zorder>Away_plus_3</zorder>
   <zorder>Away_min_1</zorder>
   <zorder>Score_reset</zorder>
   <zorder>Timer_display</zorder>
   <zorder>Timer_start</zorder>
   <zorder>Timer_pause</zorder>
   <zorder>Timer_reset</zorder>
   <zorder>Timer_set_15_minutes</zorder>
   <zorder>Timer_set_10_minutes</zorder>
   <zorder>Timer_set_5_minutes</zorder>
   <zorder>Timer_set_3_minutes</zorder>
   <zorder>Timer_set_1_minute</zorder>
   <zorder>Timer_plus_1_second</zorder>
   <zorder>Round</zorder>
   <zorder>Round_label</zorder>
   <zorder>Round_inc</zorder>
   <zorder>Round_dec</zorder>
   <zorder>Home_foul</zorder>
   <zorder>Home_foul_dec</zorder>
   <zorder>Home_foul_inc</zorder>
   <zorder>Away_foul</zorder>
   <zorder>Away_foul_inc</zorder>
   <zorder>Away_foul_dec</zorder>
   <zorder>Twenty_four_display</zorder>
   <zorder>Tet_button</zorder>
   <zorder>Home_name</zorder>
   <zorder>Away_name</zorder>
   <zorder>Twenty_four</zorder>
   <zorder>fourteen</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMusic"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBantuan">
   <property name="text">
    <string>Panduan (F1)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMusic">
   <property name="text">
    <string>Music</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
